src/js/index.js

```diff
@@ -43,7 +44,7 @@ export const hpe = deepFreeze({
       },
       icon: 'text',
       text: {
-        dark: '#FFFFFF',
+        dark: '#DDDDDD',
```

Comment:
I had held off on incorporating this change because buttons rely on `text` color. If we change this, then primary buttons go back to having the grey-ish text instead of pure white. Any thoughts on how to handle that case?

Reply:
Seems like something on the lines of this would be appropriate.
```
button: {
    color: {
      light: 'text-strong',
      dark: '#FFFFFF',
    },
  },
```
Doesn't feel right to have the published design colors and this value out of alignment, plus affects all text throughout...

Reply:
Gotcha -- I think we only want #FFFFFF if it's a primary button, but we could probably check that here. Agree we should align with published design colors but don't want a regression on the primary button color.

Reply:
addressed by maintaining primary button text color via button.extend

Reply:
Oh you're right! Good catch on latest designs.